dev-common-postgres-image/init.sql

```diff
@@ -505,8 +505,8 @@ CREATE TABLE ddl_version.version (version_tag varchar(256) PRIMARY KEY, updated_
 
 -- Version and patch number for the current ground zero,
 -- NOTE: These must be updated every time the repo is tagged
-INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (300, current_timestamp);
-INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v1.0.3', current_timestamp);
+INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (400, current_timestamp);
```

Comment:
This will clash with [this PR](https://github.com/ONSdigital/ssdc-rm-ddl/pull/118/files#diff-0266855e7bcfc8de11bbda8b5dedd94fa1387f8ac717b75f86c2b7ec1f4ab1e1R509-R510), which will probably be merged first I think